### PR TITLE
Fix Healthcheck type

### PR DESCRIPTION
**Pull request by morucci**

### CODE REVIEW

File: compose/v3/defaults.dhall [hunk 1]
```diff
@@ -61,6 +61,15 @@ let Volume =
         }
       : types.Volume
 
+let Healtcheck =
```
Comment (sbdchd) [inline compose/v3/defaults.dhall]:
```suggestion
let Healthcheck =
```

**Review by sbdchd — COMMENTED** [review]
Minor thing, looks good otherwise!

**Comment by morucci** [comment]
Thanks for the review, I've fixed it.

**Comment by sbdchd** [comment]
Thanks!

**Pull request merged**